src/NzbDrone.Core/Tv/AddSeriesService.cs

```diff
@@ -121,7 +121,7 @@ private Series AddSkyhookData(Series newSeries)
             }
             catch (SeriesNotFoundException)
             {
-                _logger.Error("TVDB ID {0} was not found, it may have been removed from TheTVDB.  Path: {1}", newSeries.TvdbId, newSeries.Path);
+                _logger.Error("Series {0} with TVDB ID {1} was not found, it may have been removed from TheTVDB.  Path: {2}", newSeries, newSeries.TvdbId, newSeries.Path);
```

Comment:
```suggestion
                _logger.Error("Series {0} with TVDB ID {1} was not found, it may have been removed from TheTVDB. Path: {2}", newSeries, newSeries.TvdbId, newSeries.Path);
```